src/main/scala/thurloe/dataaccess/SendGridDAO.scala

```diff
@@ -37,14 +37,14 @@ trait SendGridDAO {
     Note: email.setSubject and email.setText must be set even if their values
     aren't used. Supposedly this will be fixed in a future version of SendGrid
    */
-  def createEmail(toAddress: String, replyTo: Option[String], notificationId: String, substitutions: Map[String, String] = Map.empty): SendGrid.Email = {
+  def createEmail(toAddress: String, replyTos: Option[List[String]], notificationId: String, substitutions: Map[String, String] = Map.empty): SendGrid.Email = {
     val email = new SendGrid.Email()
 
     email.addTo(toAddress)
     email.setFrom(defaultFromAddress)
     email.setTemplateId(notificationId)
     email.setSubject(" ")
-    replyTo.map(email.setReplyTo)
+    replyTos.map(addrs => email.addHeader("Reply-To", addrs.mkString(", ")))
```

Comment:
`setReplyTo` unfortunately only supports one replyTo address, but you can achieve the same thing by setting the `Reply-To` header directly, which supports multiple reply-to addresses

it was a requirement that the replyTo be the admins of the group and the user requesting access